types/mongodb/index.d.ts

```diff
@@ -2481,7 +2481,7 @@ export class GridFSBucketWriteStream extends Writable {
      * @param [callback] called when chunks are successfully removed or error occurred
      * @see {@link https://mongodb.github.io/node-mongodb-native/3.6/api/GridFSBucketWriteStream.html#abort}
      */
-    abort(callback?: () => void): void;
+    abort(callback?: GridFSBucketErrorCallback): void;
```

Comment:
The callback here will not receive any data:

```suggestion
    abort(callback?: GridFSBucketErrorCallback<void>): void;
```